server/lib/nl/fulfillment/triple.py

```diff
@@ -68,16 +65,23 @@ def populate(uttr: nl_uttr.Utterance) -> bool:
     uttr.counters.err('triple_failed_no_properties', 1)
     return False
 
+  chart_added = False
   for prop in uttr.properties:
     # TODO: handle properties that are links that use ->
     if '->' in prop or '<-' in prop:
       uttr.counters.err('triple_property_unsupported', prop)
       continue
     else:
       if _populate_any_direction_prop(uttr, prop):
+        chart_added = True
         # Currently only handling one prop, so if populate succeeds for any one
         # prop, return.
-        return True
-      else:
-        uttr.counters.err('triple_property_failed_existence', prop)
-  return False
+        break
+  # If we are populating for a single entity and we've added an answer for this
+  # entity, also add an entity overview chart
+  if chart_added and len(uttr.entities) == 1:
```

Comment:
Even if `chart_added` is false, does it make sense to show ENTITY_OVERVIEW (with a message about property not existing)?

Feel free to add TODO for follow up, if you agree.

Reply:
I think this is something we will need to revisit because if we eventually interleave entity/prop charts with the regular sv charts, I don't think it would make sense to add the entity overview here. Rather, we might want to add the entity overview at the very end of all the charts? Added a TODO to revisit